lib/release-notes-status-bar.coffee

```diff
@@ -4,9 +4,14 @@
 module.exports =
 class ReleaseNotesStatusBar extends View
   @content: ->
-    @span type: 'button', class: 'release-notes-status icon icon-squirrel inline-block'
+    # 'blank' top level span due to issue #58
+    @span =>
+      @span type: 'button', class: 'release-notes-status icon icon-squirrel inline-block', outlet: 'statusIcon'
```

Comment:
You can remove the outer span and remove the `outlet:` value and then add the class down below by doing:


```
@addClass 'release-notes-status-available'`
```

instead of ` @statusIcon.addClass 'release-notes-status-available'`

You can't have a top level outlet name but the top level element is always available as `@`.